liota/core/package_manager.py

```diff
@@ -482,17 +477,10 @@ def run(self):
 
             # Switch on message content (command), determine what to do
             command = msg[0]
-            if command in ["unload", "delete", "list", "stat", "unload_all",\
-                           "terminate"]:
-                # currently, write 'Success' to the named pipe once
-                #     successfully receives these commands
-                self._write_cmd_exec_status('Success\n')
-
-            # currently, write 'Success' to the named pipe after verification
-            #     of the checksum for load/reload/update related commands
             if command in ["load", "reload", "update"]:
                 #-----------------------------------------------------------
                 # Use these commands to handle package management tasks
+
```

Comment:
Remove the extra line, reformat all the files using autopep format.

Reply:
This is a revert of a commit. I don't want to introduce changes in a revert. Ideally, each developer should take care of formatting during initial check-in itself or there should be tests in the code base which fail if there are formatting issues.

Reply:
Now the changes are getting introduced in form of code revert and formatting should be done.

Reply:
As discussed, formatting and all should not be taken care of as part of a git revert. It should ideally just revert whatever was introduced,